reg/drone/physics/electricity/Source.0.1.uavcan

```diff
@@ -0,0 +1,27 @@
+# A generic source or sink of electric power (battery, turbogenerator, braking resistor, etc.).
+# Low-pass filtering should be applied to avoid aliasing effects (as is the case everywhere else).
+
+Power.0.1 load_power
+# Total instant power supplied to the load.
+# Positive current flows from the source to the load (discharge/production).
+# Negative current flows into the source (regeneration/consumption).
+
+uavcan.si.unit.energy.Scalar.1.0 available_energy
+# A pessimistic estimate of the amount of energy that can be reclaimed from the source in its current state.
```

Comment:
Why pessimistic?

Reply:
Because the interface provides information about the energy that can be reclaimed regardless of the amount of leftover (unused) energy. If you have 100 J, any value from [0, 100] J is a technically valid estimate because that much energy can be retrieved from the source. We could provide a specific confidence interval here but I suppose in the MVP we can omit that level of detail.